hack/local-up-cluster.sh

```diff
@@ -112,6 +114,11 @@ if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
     fi
 fi
 
+# warn if users are running with swap allowed
+if [ "${FAIL_SWAP_ON}" == "false" ]; then
+    echo "WARNING : The kubelet is configured to not fail if swap is enabled; production deployments should disable swap." 2>&1
```

Comment:
Did you mean to redirect stderr to stdout? Or was this meant to be the other way?

Reply:
I missed that when copy/paste from another echo stmt.  will update to just do a normal echo.